### PR TITLE
docs/remote.md: Remove unnecessary backquote

### PR DESCRIPTION
**Pull request by npcode**
Signed-off-by: Yi EungJun eungjun.yi@navercorp.com

### CODE REVIEW

File: docs/remote.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ When the proxy is asked to create a network, the remote process shall receive a
 
 The response indicating success is empty:
 
-    `{}`
+    {}
```
Comment (aboch) [inline docs/remote.md]: Can you please also take care of the same at line 260 and 290.
Thanks.

Comment (npcode) [inline docs/remote.md]: Okay, I've done.

**Comment by aboch** [comment]
Thanks.
LGTM

**Comment by aboch** [comment]
CI failure is because remote branch does not have a fix for the CI. CI is irrelevant for this change.

**Pull request merged**